OWNERS

```diff
@@ -0,0 +1,24 @@
+reviewers:
+  - abhat
+  - alexanderConstantinescu
+  - aojea
+  - danielmellado
+  - danwinship
+  - dcbw
+  - JacobTanenbaum
+  - juanluisvaladas
+  - rcarrillocruz
+  - squeed
+  - tssurya
+  - trozet
+approvers:
+  - abhat
+  - danwinship
+  - dcbw
+  - knobunc
+  - trozet
+  - rcarrillocruz
+  - squeed
+
+# Bugzilla info
+component: Networking
```

Comment:
I think you need to specify subcomponent as well. Even though there isn't a correct answer. Pick either openshift-sdn or ovn-kubernetes

Reply:
Done